src/utils/ConfigLoader.ts

```diff
@@ -0,0 +1,20 @@
+import * as dotenv from 'dotenv';
+
+const DEFAULT_ENV_FILE = `${__dirname}/../../.env`;
+
+const loadConfigHelper = () => {
```

Comment:
I'd change that name as I am not sure what helper means in that context. Do you mean that config object returned by `dotenv.config` is a helper? Or is that method a helper method?